libchannelserver/ChannelRPCServer.cpp

```diff
@@ -1147,17 +1148,15 @@ dev::channel::TopicChannelMessage::Ptr ChannelRPCServer::pushChannelMessage(
 
         if (!response)
         {
-            throw dev::channel::ChannelException(99, "send fail, all retry failed");
+            BOOST_THROW_EXCEPTION(ChannelException(99, "send fail, all retry failed"));
         }
 
         return response;
     }
     catch (dev::channel::ChannelException& e)
     {
-        CHANNEL_LOG(ERROR) << "pushChannelMessage error"
-                           << LOG_KV("what", boost::diagnostic_information(e));
-
-        throw e;
+        CHANNEL_LOG(ERROR) << "pushChannelMessage error" << LOG_KV("what", e.what());
```

Comment:
how about `e.what()`->`boost::diagnostic_information` for detailed information.

Reply:
because it is useless, output the same as e.what(). the difference is diagnostic make log untidy